packages/face-detection/src/definitions.ts

```diff
@@ -12,18 +12,29 @@ export interface FaceDetectionPlugin {
 /**
  * @since 5.1.0
  */
-export interface ProcessImageOptions {
+export type ProcessImageOptions = (
+  | {
+      /**
+       * The local path to the image file.
+       *
+       * @since 5.1.0
+       */
+      path: string;
+    }
+  | {
+      /**
+       * Represents an image object (Base64 encoded).
+       *
+       * @since 5.1.0
+       */
+      image: string;
```

Comment:
I would like to remove this option. This feature can lead to OOM exceptions. You should always try to use a native file path. If you really only have the image available in Base64 format in the webview, then you can still write it to the filesystem using the Capacitor Filesystem plugin (or the Capacitor Blob Writer plugin). But this prevents issues with this plugin regarding OOM.